terraform/fec_rds.tf

```diff
@@ -105,6 +105,11 @@ resource "aws_db_parameter_group" "fec_default" {
         name  = "log_statement"
         value = "ddl"
     }
+
+    parameter {
+        name  = "max_standby_streaming_delay" # This has no effect on masters, it only affects slaves
+        value = "1 h"
```

Comment:
The discussion settled on an hour gives a big buffer to keep queries moving. Do you have any concerns with it being too long or short?